test/controllers/articles_controller_test.rb

```diff
@@ -3,28 +3,86 @@
 require "test_helper"
 
 # Controller tests for news articles
-class ArticleControllerTest < FunctionalTestCase
-  def test_create_article_get
+class ArticlesControllerTest < FunctionalTestCase
+  ############ test Actions that Display data (index, show, etc.)
```

Comment:
As with the controller, I suggest ordering test methods uniformly, and in a way that mirrors the order of controller methods.